modules/programs/jb-remote.nix

```diff
@@ -0,0 +1,36 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.programs.jb-remote;
+
+  mkLine = ide:
+    # Errors out if the symlink already exists
+    "${ide}/bin/${ide.meta.mainProgram}-remote-dev-server registerBackendLocationForGateway || true";
+  lines = map mkLine cfg.ides;
+  linesStr = ''
+    rm $HOME/.cache/JetBrains/RemoteDev/userProvidedDist/_nix_store*
+  '' + (concatStringsSep "\n" lines);
+  dagEntry = hm.dag.entryAfter [ "writeBoundary" ] linesStr;
+
+in {
+  meta.maintainers = with lib.maintainers; [ genericnerdyusername ];
+
+  options.programs.jb-remote = {
```

Comment:
I don't think we have any abbreviations elsewhere, so we should probably say `jetbrains-remote` or similar instead of `jb-remote` .